vshard.go

```diff
@@ -163,6 +163,10 @@ type InstanceInfo struct {
 	// While this information is not mandatory, it can be useful for internal management or tracking purposes.
 	// The UUID ensures that each instance can be identified uniquely, but it is not required for basic operations.
 	UUID uuid.UUID
+
+	// Allows to use custom dialer for instance connection.
```

Comment:
I guess this comment would be more informative.
`Dialer allows to use a custom dialer instead of the default one (tarantool.NetDialer)`